dcos_installer/config.py

```diff
@@ -45,6 +45,13 @@ def normalize_config_validation(messages):
     return validation
 
 
+def normalize_config_validation_exception(exception):
+    messages = {}
+    messages['errors'] = exception.errors
```

Comment:
As discussed - this won't work and it should be something like `exception.errors`.

I'd like it if a test in this repository would catch that but I'm happy to discuss the trade-offs involved there.

Reply:
Ah! Race condition between comment and change 😄